### ISSUE TITLE
Test accuracy reduced to 0%, but the train accuracy remains high.

### ISSUE THREAD

**Issue by dezengzang** [liuzechun/ReActNet#9]
When I run [resnet/2_step2/run.sh](https://github.com/liuzechun/ReActNet/blob/master/resnet/2_step2/run.sh) for 132 epochs, I find that the test accuracy reduces to 0, but the train accuracy remains high. This situation does not happen before epoch131, have you encountered a similar situation? I only changed the batch size to 320. Thanks.
```
Epoch: [132][4002/4004] Time  0.462 ( 0.522)    Data  0.000 ( 0.062)    Loss 1.9292e+00 (1.8380e+00)    Acc@1  59.38 ( 59.75)   Acc@5  80.94 ( 80.55)
Epoch: [132][4003/4004] Time  0.325 ( 0.522)    Data  0.000 ( 0.062)    Loss 2.1053e+00 (1.8380e+00)    Acc@1  56.52 ( 59.75)   Acc@5  77.29 ( 80.55)
Test: [  0/157] Time 14.804 (14.804)    Loss 2.3422e+01 (2.3422e+01)    Acc@1   0.00 (  0.00)   Acc@5   0.00 (  0.00)
Test: [  1/157] Time  0.189 ( 7.496)    Loss 2.6162e+01 (2.4792e+01)    Acc@1   0.00 (  0.00)   Acc@5   1.88 (  0.94)
Test: [  2/157] Time  0.184 ( 5.059)    Loss 2.4110e+01 (2.4565e+01)    Acc@1   0.31 (  0.10)   Acc@5   0.94 (  0.94)

```

**Comment by liuzechun** [comment]
Hi, Thanks for your question. I didn't encounter a similar problem in my experiments. When you reduce the batch size from 512 to 320, did you linearly reduce the initial learning rate from 1e-3 to 6.25e-4? Because the leraning rate being too large will make the network diverge.
